File: assignment-18/task4.js
/**
 * Takes an array of student names and prints each name to the console.
 * @param {string[]} students - An array of student names.
 */
function printStudents(students) {
  console.log("Student List:");
  students.forEach(student => {
    console.log(student);
  });
}

// Sample data
const studentNames = ["Alice", "Bob", "Charlie"];

// Test the function
printStudents(studentNames);

File: assignment-18/task4.py
def print_students(students: list[str]) -> None:
    """
    Takes a list of student names and prints each name.
    
    Args:
        students: A list of student names.
    """
    print("Student List:")
    for student in students:
        print(student)

# Sample data
student_names = ["Alice", "Bob", "Charlie"]

# Test the function
print_students(student_names)
